step.yml

```diff
@@ -24,20 +24,76 @@ toolkit:
   go:
     package_name: github.com/bitrise-steplib/bitrise-step-xcode-start-simulator
 inputs:
-  - destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
-    opts:
-      title: Device destination specifier
-      summary: Destination specifier describes the simulator device to be started.
-      description: |-
-        Destination specifier describes the simulator device to be started.
+- destination: platform=iOS Simulator,name=iPhone 8 Plus,OS=latest
+  opts:
+    title: Device destination specifier
+    summary: Destination specifier describes the simulator device to be started.
+    description: |-
+      Destination specifier describes the simulator device to be started.
 
-        The input value uses the same format as xcodebuild's `-destination` option.
-      is_required: true
+      The input value uses the same format as xcodebuild's `-destination` option.
+    is_required: true
+
+- erase: "no"
```

Comment:
Do have clean simulators AFAIK.
It could be only needed when running multiple tests on the same simulator (as it also shutdowns it, besides erasing). It is providing peace of mind that we have this option rather then a strictly required one.